flexmeasures/api/v3_0/tests/test_assets_api.py

```diff
@@ -83,6 +101,9 @@ def test_get_assets(
     account_name,
     num_assets,
     use_pagination,
+    sort_by,
+    sort_dir,
+    expected_asset_name,
```

Comment:
Better name for parameter: "expected_name of_first_asset"

Reply:
ok, noted